packages/components/src/Avatar/Avatar.module.css

```
@@ -0,0 +1,146 @@
+.wrapper {
+  --avatar-x-y: 1.5rem;
+
+  background: var(--color-gray-300);
+  border-radius: 100%;
+  box-sizing: border-box;
+  overflow: hidden;
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  width: var(--avatar-x-y);
+  height: var(--avatar-x-y);
+  box-shadow: var(--shadow-small-box-shadow);
+  border: 3px solid var(--color-white);
+}
+
+.wrapper.small {
```

Haha oh no! I must not have been clear enough. Here's an example:

```suggestion
.small {
```

---

😭